### DISCUSSION TITLE
can It work with Jetson Nano devkit (B01)

### DISCUSSION THREAD

**Discussion by thanhtruongphan** [pythops/jetson-image#143] [General]
Hello everyone, have a nice day~ 

I'm new to Jetson, I'm looking for Ubuntu 22.04 (for a project using ROS2-humble) that works with my Jetson nano Developer kit (B01) and has JetPack (to do with image-processing)

Is this the repo I'm looking for?
Help me, pls... 

- thank you -

**Comment by pythops** [comment]
This is the minimalist image, so you will need to install all the necessary packages later for your project, plus there is no GUI initially, so everything will be done in the terminal. If you are not comfortable with that, then you should stick with the official image from Nvidia.
